reflred/scale.py

```diff
@@ -79,7 +79,7 @@ def apply_norm(data, base='auto'):
         M = data.monitor.count_time
         M[M == 0] = data.monitor.time_step/2.  # protect against zero count time
         # Uniform distribution has variance of interval width/12
-        varM = data.monitor.time_step/12.
+        varM = (data.monitor.time_step**2)/12.
```

Comment:
Comment is wrong as well.

Reply:
Didn't catch your comment in time... I'll just change the comment in a regular push.